File: sudokuUtils.java
public class sudokuUtils {
    
    protected static void printSudoku(int[][] grid) {
        for (int row = 0; row < 9; row++) {
            for (int col = 0; col < 9; col++) {
                System.out.print(grid[row][col] + " ");
            }
            System.out.println();
        }
    }
}

File: sudokuUtils.js
class sudokuUtils {
    static printSudoku(grid) {
        for (let row = 0; row < 9; row++) {
            for (let col = 0; col < 9; col++) {
                process.stdout.write(grid[row][col] + " ");
            }
            console.log();
        }
    }
}

module.exports = sudokuUtils;